Fe 8 event list

dominikapap opened the pull request.

abdevworks committed:
feat: side-menu

Initial setup for side-menu

abdevworks committed:
Merge branch 'develop' of https://github.com/akademia-envelo-3/MELO-front into fe-7-create-side-menu

abdevworks committed:
feat: side-menu

Added basic menu for mobile

abdevworks committed:
feat: side-menu

Basic setup for active sub category item

abdevworks committed:
Merge branch 'develop' of https://github.com/akademia-envelo-3/MELO-front into fe-7-create-side-menu

abdevworks committed:
feat: side-menu

Added active color change for sub categories. Category icons change order based on breakpoint

abdevworks committed:
feat: side-menu

Code rafactor and testing how it looks with other components

abdevworks committed:
feat: side-menu

Before desktop version addded

abdevworks committed:
feat: side-menu

Trying to fix side menu view

abdevworks committed:
Merge branch 'develop' of https://github.com/akademia-envelo-3/MELO-front into fe-7-create-side-menu

LukaszStaniszewski committed:
feat: reinstalled libraries

abdevworks committed:
feat: side-menu

Created basic side menu for desktop

abdevworks committed:
feat: side-nav

Fixed display bug for desktop version

LukaszStaniszewski committed:
Merge branch 'develop' of https://github.com/akademia-envelo-3/MELO-front into develop

abdevworks committed:
feat: side-menu

Code refactor and fixed bugs with menu display

abdevworks committed:
Merge branch 'develop' of https://github.com/akademia-envelo-3/MELO-front into fe-7-create-side-menu

abdevworks committed:
feat: side-menu

Fixing element styling

abdevworks committed:
feat: side-menu

Fixed closing when you click outside of side menu container

abdevworks committed:
feat: side-menu

Refactored version of side menu

abdevworks committed:
feat: side-menu

Fixed side menu animation on window resize

abdevworks committed:
feat: side-menu

Added option to open side menu on mobile devices from navbar

LukaszStaniszewski committed:
feat: created event list component, added initial styles and html

LukaszStaniszewski committed:
Merge branch 'fe-7-create-side-menu' of https://github.com/akademia-envelo-3/MELO-front into Fe-8-event-list

LukaszStaniszewski committed:
style: add styles

LukaszStaniszewski committed:
Merge branch 'develop' of https://github.com/akademia-envelo-3/MELO-front into develop

LukaszStaniszewski committed:
Merge branch 'Fe-8-event-list' of https://github.com/akademia-envelo-3/MELO-front into Fe-8-event-list

LukaszStaniszewski committed:
feat: cards are loaded from mocked data base

LukaszStaniszewski committed:
feat: added fetch of cards limited to 12 items, improved styles, added inject hook

LukaszStaniszewski committed:
fix: event card is not breaking with longer descriptions

LukaszStaniszewski committed:
feat: removed unnecessary imports

dominikapap committed:
fix: card overflow

changed max card with to 100vw to prevent overflow

dominikapap committed:
refactor: side menu

removed side menu since it's not in develop yet

dominikapap committed:
refactor: unused code

removed unused code

dominikapap committed:
refactor: event list

minor changes

dominikapap committed:
refactor: unused import

removed unused import

dominikapap committed:
fix: package.json

removed accidental change

dominikapap committed:
style: card component

added separate file for styles

dominikapap committed:
style: card styles

moved styles to a separate folder

rafalfuz commented on src/app/features/event/event.module.ts:
```diff
@@ -1,14 +1,27 @@
-import { DatePipe, NgClass, NgIf, UpperCasePipe } from '@angular/common';
-
-import { PowerIconComponent } from '@shared/ui';
+import { CommonModule, DatePipe, NgClass, NgIf, UpperCasePipe } from '@angular/common';
```
.

dominikapap replied:
poprawione

abdevworks commented on src/app/features/event/event.module.ts:
```diff
@@ -1,14 +1,27 @@
 import { MatIconModule } from '@angular/material/icon';
-import { CardComponent } from '.';
+import { CardComponent, EventListComponent } from '.';
+import { PowerIconComponent } from './power-icon/power-icon.component';
```
Tutaj też można zmienić na import { PowerIconComponent } from '.'; Więc ostatecznie: import { CardComponent, EventListComponent, PowerIconComponent } from '.';

dominikapap replied:
poprawione

abdevworks reviewed (approved):
Sprawdzone.

dominikapap committed:
fix: duplicated imports, service providers

removed dupliated imports in event module, changed providers for event api service

dominikapap committed:
Merge branch 'develop' into fe-8-event-list
